styles/globals.css

```diff
@@ -18,9 +18,11 @@
     --nav-background: #080808;
     --nav-height: 6rem;
     --dashboard-nav-width: 15rem;
-    --footer-height: 4rem;
+    --footer-height: 10rem;
 
     --hover-transition: 0.25s ease-in-out;
+
+    --brand-border-1: 3px solid var(--brand-color-1a);
```

Comment:
Did you add the new pink and reorder the brand-color variables to their new values?

Reply:
yea replaced with 
`    --brand-color-1: #ffcfcf;
    --brand-color-2: #c8e1d5;
    --brand-color-3: #ffe7d3;`